### DISCUSSION TITLE
prisma deployed to azure functions app error Query engine binary

### DISCUSSION THREAD

**Discussion by diogomadeira-dev** [prisma/prisma#16706] [Q&A] [answered]
`Invalid `prisma.user.create()` invocation in
C:\home\site\wwwroot\CreateUser\index.js:6:36

  3 module.exports = async function (context, req) {
  4   const { email, name } = req.body
  5   try {
→ 6     const user = await prisma.user.create(
Query engine binary for current platform "windows" could not be found.
This probably happens, because you built Prisma Client on a different platform.
(Prisma Client looked in "C:\home\site\wwwroot\node_modules\@prisma\client\runtime\query-engine-windows.exe")

Searched Locations:

  C:\home\site\wwwroot\node_modules\.prisma\client
  C:\Users\Diogo\Desktop\prisma-azure\node_modules\@prisma\client
  C:\home\site\wwwroot\node_modules\@prisma\client
  C:\home\site\wwwroot\node_modules\.prisma\client
  C:\home\site\wwwroot\prisma
  /tmp/prisma-engines
  C:\home\site\wwwroot\node_modules\.prisma\client

You already added the platforms "windows", "windows" to the "generator" block
in the "schema.prisma" file as described in https://pris.ly/d/client-generator,
but something went wrong. That's suboptimal.

Please create an issue at https://github.com/prisma/prisma/issues/new`

**Comment by nurul3101** [comment, marked as answer]
Hello @diogomadeira1006 👋 

Can you try including [binaryTarget](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#binarytargets-options) option as "native" in your schema file? 

```prisma
generator client {
  provider      = "prisma-client-js"
  binaryTargets = ["native"]
}
```
Let me know if you still get an error.

**Reply by pantharshit00** [reply]
👋 Hello @diogomadeira1006,  did you perhaps have a chance to check @nurul3101's recommendation? Let us know if you still have any questions!
 
Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!
